gradle.properties

```diff
@@ -10,7 +10,7 @@ pluginSinceBuild=232
 #pluginUntilBuild=233.*
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType=IC
-platformVersion=2023.2
+platformVersion=2024.1.4
```

Comment:
nope

Reply:
I know about that, I wanted just to check that test is working with 2024.1.4 andit seems it fails on Mac OS. I redo a new fix, if CI build is working I will reswitch to 2023.2